pkg/storage/ingest/pusher.go

```diff
@@ -122,7 +122,7 @@ func (c pusherConsumer) pushToStorage(ctx context.Context, tenantID string, req
 			if reason != "" {
 				err = fmt.Errorf("%w (%s)", err, reason)
 			}
-			level.Warn(spanLog).Log("msg", "detected a client error while ingesting write request (the request may have been partially ingested)", "err", err, "user", tenantID)
+			level.Warn(spanLog).Log("msg", "detected a client error while ingesting write request (the request may have been partially ingested)", "user", tenantID, "insight", "true", "err", err)
```

Comment:
```suggestion
			level.Warn(spanLog).Log("msg", "detected a client error while ingesting write request (the request may have been partially ingested)", "user", tenantID, "insight", true, "err", err)
```

Reply:
In https://github.com/grafana/mimir/blob/main/pkg/ruler/compat.go#L327C72-L327C79 we use `true` instead of `true` the insights logs pickup might cater both (I don't know where the code for picking them up lives to double check). But with `true` is at least consistent.

Reply:
Sounds good. It doesn't really matter -- simple strings are logged without quote marks -- but let's be consistent.